secretstemplate

```diff
@@ -1,21 +1,27 @@
-# pingone terraform worker app Credentials
 export TF_VAR_pingone_client_id=''
 export TF_VAR_pingone_client_secret=''
-export TF_VAR_pingone_environment_id=''
-export TF_VAR_pingone_region=''
+# one of: "AP | AU | CA | EU | NA"
+export TF_VAR_pingone_client_region_code=''
+export TF_VAR_pingone_client_environment_id=''
 export TF_VAR_pingone_license_id=''
-# davinci terraform admin user credentials
-export TF_VAR_pingone_username=''
-export TF_VAR_pingone_password=''
-export TF_VAR_pingone_davinci_environment_id=''
-# group_id for davinci environment terraform user accounts
+export TF_VAR_pingone_davinci_admin_username=''
+export TF_VAR_pingone_davinci_admin_password=''
+export TF_VAR_pingone_davinci_admin_environment_id=''
+# one of: "NorthAmerica | Canada | Asia | Europe"
+export TF_VAR_pingone_davinci_admin_region=''
+export TF_VAR_pingone_davinci_user_group_id=''
 export TF_VAR_pingone_davinci_terraform_group_id=''
+export TF_VAR_pingone_environment_type=""
```

Comment:
should we give the "one of: comment to guide the user here?  I think it'll be either SANDBOX or PRODUCTION, right?

Reply:
done